pwndbg/disasm/color.py

```diff
@@ -25,7 +28,7 @@ def instruction(ins):
 
         # If it's a constant expression, color it directly in the asm.
         if const:
-            asm = asm.replace(hex(ins.target), sym or target)
+            asm = asm.replace(hex(ins.target), maybe_highlight(sym or target))
```

Comment:
the thing is that some part that adds colors by itself or make something bold also resets the color codes no normal so i will need to make that check on multiple locations, thats why i tried to just pass a bool flag to the function.
I removed that bool now, we can review the new patch once I tested it a bit, shouldn't take too long